### ISSUE TITLE
all info generated by Rest Server script should be consistent with Launcher returned info

### ISSUE THREAD

**Issue by yqwang-ms** [Microsoft/pai#426]
[ERROR] ContainerList
0-0-10.151.40.183-14105
0-0-10.151.40.204-17084
2 containers are available, not equal to 1, exit ...

In the first attempt,
AM start container_e18_1522398176159_2247_01_000002.
But it then failed to store it in ZK due to ZK connection failure.

ZK failure is by design acceptable, so we start second attempt,
And the new AM start container_e18_1522398176159_2247_02_000002, since it does not know container_e18_1522398176159_2247_01_000002.

AM will kill the container_e18_1522398176159_2247_01_000002, since it is unmanaged by it any more.
But before kill it, Yifan’s script detect there are 2 running container, and just exit the container_e18_1522398176159_2247_02_000002.
So, AM stopped the whole job.

In short words, it is an inconsistent between Yifan’s script and Launcher.
2 options to fix it:
1.	Remove the assert.
2.	Assert the IP/Host only for these in Launcher returned FrameworkStatus.
